### ISSUE TITLE
Client cannot connect to rogue access point, simply disconnects.

### ISSUE THREAD

**Issue by ock666** [wifiphisher/wifiphisher#1446]
Hoping somebody could provide some insight into why I'm running into troubles.

have tried installed wifiphisher from both source and APT with the same issues.

I have two wireless interfaces capable of monitor mode and packet injection as well.
(RTL8188FU & a broadcom chipset (unsure of model number)

I have tried running wifiphisher with the -nD flag to see if it was possible that the client was associating to the AP but receiving deauth frames and being bumped from the network.

I have tried using other devices to connect to the AP, some devices associate for a second and then drop. Others connect but do not recieve internet or the specified template/attack.

problem also persists when attempting to use airbase-ng to generate the AP.

**Comment by devilmen80** [comment]
Yes i said this problem some months ago...and the author said me reinstall the program. Again i have your same problem...

**Comment by devilmen80** [comment]
Ok i found solution mate.
If you have firewall you must disable it to reset iptables also before to run attack with wifiphisher.
I hope solution will work with you also.
Regards.

**Comment by sophron** [comment]
@devilmen80 Can you elaborate? What firewall did you have and how did you disable it?

**Comment by devilmen80** [comment]
Hello mate i had ufw enable on the system...so  i disable it with: 
1)sudo ufw reset and sudo ufw disable. 
Now look at iptables why you could need to reset your rules.
Anyway refresh them with:
sudo iptables -F
sudo iptables -X
At this point see for iptables --list and all should be back normally.
If are present rules for iptables legacy you will have to reset even them with same process.
Now wifiphisher will work or at least it was with me.
Regards

**Comment by ock666** [comment]
Okay so I figured out what the issue I was facing.
When installing from apt it seems I had missed some dependencies which I believe from memory was an ssl library and something else.

I solved my issue by:
installing wifiphisher from source
installing any dependencies which get thrown in errors as you're building/installing
occasionally the problem with the client being unable to connect would arise,
however I solved this by starting and stopping dnsmasq
and running wifiphisher again.
